README.md

```diff
@@ -440,6 +443,7 @@ To start your Astro project, follow the usual practice:
 ```bash
 cd my-astro-project
 npm install
+export APOS_EXTERNAL_FRONT_KEY=your-secret-goes-here
```

Comment:
For now this should help a great deal. I would like to do more in terms of normalizing practices around environment variables but I'm getting different input from you and Bob on how Astro likes to do it, we have thoughts on how Apostrophe likes to do it, etc. Leaving this as a decision for the developer is reasonable for this week.